site/content/contribute/webapp/end-to-end-tests.md

```diff
@@ -36,32 +46,29 @@ The folder structure is mostly based on the [Cypress scaffold](https://docs.cypr
   |-- package.json
 ```
 
-- `/e2e/cypress/fixtures` or [Fixture Files](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Fixture-Files):
+1. `/e2e/cypress/fixtures` or [Fixture Files](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Fixture-Files):
     - Fixtures are used as external pieces of static data that can be used by tests.
     - Typically used with the `cy.fixture()` command and most often when stubbing Network Requests.
-- `/e2e/cypress/integration` or [Test Files](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Test-files)
+2. `/e2e/cypress/integration` or [Test Files](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Test-files)
     - To start writing tests,
         - simply create a new file (e.g. `login_spec.js`) within `/e2e/cypress/integration` folder and;
         - refresh tests list in the Cypress Test Runner and a new file should have appeared in the list.
     - Subfolder naming convention depends on test grouping, which is usually based on the name of tab in the Release Testing spreadsheet (e.g. `/e2e/cypress/integration/messaging/` for "Messaging" tab).
```

Comment:
This may be out of scope for these updates, but should we replace `name of tab in the Release Testing spreadsheet` with something like `general functional area`?
```suggestion
    - Subfolder naming convention depends on test grouping, which is usually based on the general functional area (e.g. `/e2e/cypress/integration/messaging/` for "Messaging" tab).
```